foreman_discovery.gemspec

```diff
@@ -20,5 +20,5 @@ Gem::Specification.new do |s|
   s.licenses = ["GPL-3"]
   s.summary = %q{MaaS Discovery Plugin for Foreman}
 
-  s.add_dependency 'deface', '< 1.0'
+  s.add_dependency 'deface'
```

Comment:
I think most (all?) of us limited to '< 2.0': https://github.com/theforeman/foreman_salt/pull/42, https://github.com/theforeman/foreman_openscap/pull/114/files

Which makes sense, if they end up releasing 2.0 with incompatibilities, this might break.